.github/workflows/release.yml

```diff
@@ -17,8 +17,9 @@ jobs:
       - name: build
         run: |
           version=`cat VERSION`
-          go build -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
-
+          go build -o pulsarctl.sh -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
```

Comment:
why the output is `pulsarctl.sh`?

Reply:
because it is conflict with the output of Darwin. I need a different name.

Reply:
Maybe pulsarctl-linux is more better?